spec/04-status_api_spec.lua

```diff
@@ -383,8 +383,8 @@ describe("Plugin: prometheus (access via status API)", function()
     })
     local body = assert.res_status(200, res)
     assert.matches('kong_memory_lua_shared_dict_total_bytes' ..
-                   '{shared_dict="prometheus_metrics"} 5242880', body, nil, true)
+                   '{shared_dict="prometheus_metrics",kong_subsystem="http"} 5242880', body, nil, true)
     assert.matches('kong_memory_lua_shared_dict_bytes' ..
-                   '{shared_dict="prometheus_metrics"}', body, nil, true)
+                   '{shared_dict="prometheus_metrics",kong_subsystem="http"}', body, nil, true)
```

Comment:
Can we please add a test for an SHM in `stream` sub-system?

Reply:
added